docs/developer/getting-started.md

```diff
@@ -21,11 +21,24 @@ We will assume that you have a UNIX compatible machine at your disposal (Mac, Li
 If not done already please install these required tools
 
 - [Docker Desktop](https://www.docker.com/products/docker-desktop/) (if using Mac or Windows), on Linux you should be good with installing the docker package.
-- [Node](https://nodejs.org/en/download) version 20 or above with `nvm` (recommend)
+- [Node](https://nodejs.org/en/download) version 20 or above with `volta` or `nvm` (recommend, see the [intro section](./intro#required))
```

Comment:
Set an [explicit ID](https://docusaurus.io/docs/markdown-features/toc#heading-ids) in the intro to avoid broken links in the future. The syntax is 
```markdown
### Hello World {#my-explicit-id}
```

Reply:
Ah good to know !